src/NServiceBus.AcceptanceTests/Serialization/When_skip_wrapping_xml.cs

```diff
@@ -0,0 +1,92 @@
+﻿namespace NServiceBus.AcceptanceTests.Serialization
+{
+    using System.Linq;
+    using System.Text;
+    using System.Threading.Tasks;
+    using System.Xml.Linq;
+    using AcceptanceTesting;
+    using EndpointTemplates;
+    using MessageMutator;
+    using NUnit.Framework;
+
+    public class When_skip_wrapping_xml : NServiceBusAcceptanceTest
```

Comment:
urgh, sorry I messed the tests up in my head. Yeah that could probably make sense.